test/parallel/test-http-parser-free.js

```diff
@@ -42,15 +44,13 @@ server.listen(0, function() {
           assert.strictEqual(req.parser, parser);
         }
 
-        if (++responses === N) {
-          server.close();
-        }
+        countdown.dec();
         res.resume();
       });
     })(i);
   }
 });
 
 process.on('exit', function() {
```

Comment:
Do we still need this check? Doesn't `server.close()` / end of execution from the `Countdown` mean we've already hit zero remaining?

cc @jasnell

Reply:
Yep, the `process.on('exit')` can be removed.